core/tests/phases_test.py

```diff
@@ -1,20 +1,22 @@
 import unittest
 from core.game import Game
-from core.phases import SelectFoodAndClimatePhase, PhaseState
+from core.phases import PhaseState, SelectFoodAndClimatePhase
 from core.trait_card import TraitCard
 
 
 class TestSelectFoodAndClimatePhase(unittest.TestCase):
     def setUp(self):
-        self.game = Game(["Smith", "Gould"])
-        self.phase = SelectFoodAndClimatePhase(self.game)
+        self._game = Game(["Smith", "Gould"])
```

Comment:
Think this might have been an intended refactor? Although to be honest we could make all such members private? What do you think?

Reply:
Nice spot.